repository-data/webfiles/src/main/resources/site/freemarker/hee/layout/base-top.ftl

```diff
@@ -2,75 +2,100 @@
+            <a class="nhsuk-header__link" href="<@hst.link siteMapItemRefId='root'/>" aria-label="${hstRequestContext.resolvedMount.mount.channelInfo.organisationName}">
+                <#if hstRequestContext.resolvedMount.mount.channelInfo.organisationDescriptor?has_content>
                 <span class="nhsuk-organisation-descriptor">
                     ${hstRequestContext.resolvedMount.mount.channelInfo.organisationDescriptor}
                 </span>
-            </#if>
-            <#--  Base org name & descriptor: END  -->
-        </a>
+                </#if>
+            </a>
+            <#--  Channel org name & descriptor: END  -->
+        </span>
+        <#--  NHS/HEE logo and org name & descriptor: END  -->
+
+        <#--  Multi org logo  -->
+        ${multiOrgLogoHTML}
+
     </div>
-    <#--  Base logo and org name & descriptor: END  -->
+    <#-- Logos: END -->
+
+    <#-- Search + Toggles: START -->
+    <#assign hasSearch=true>
+    <#assign hasNav=true>
 
-    <div class="nhsuk-header__content" id="content-header">
-        <div class="nhsuk-header__menu">
-            <button class="nhsuk-header__menu-toggle" id="toggle-menu" aria-controls="header-navigation" aria-label="Open menu">Menu</button>
+    <#if item.hasSearch??>
+        <div class="nhsuk-header__content" id="content-header">
+            <#if item.hasNav??>
+            <div class="nhsuk-header__menu">
+                <button class="nhsuk-header__menu-toggle" id="toggle-menu" aria-controls="header-navigation" aria-label="Open menu">Menu</button>
+            </div>
+            </#if>
+            <div class="nhsuk-header__search">
+                <button class="nhsuk-header__search-toggle" id="toggle-search" aria-controls="search" aria-label="Open search">
+                    <svg class="nhsuk-icon nhsuk-icon__search" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
+                        <path d="M19.71 18.29l-4.11-4.1a7 7 0 1 0-1.41 1.41l4.1 4.11a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42zM5 10a5 5 0 1 1 5 5 5 5 0 0 1-5-5z"></path>
+                    </svg>
+                    <span class="nhsuk-u-visually-hidden">Search</span>
+                </button>
+                <div class="nhsuk-header__search-wrap" id="wrap-search">
+                    <@hst.link var="searchLink" siteMapItemRefId="search"/>
+                    <form class="nhsuk-header__search-form" id="search" action="${searchLink}" method="get" role="search">
+                        <label class="nhsuk-u-visually-hidden" for="search-field">Search the NHS website</label>
+                        <div class="autocomplete-container" id="autocomplete-container"></div>
+                        <input class="nhsuk-search__input" id="search-field" name="{{ params.searchInputName if params.searchInputName else 'q' }}" type="search" placeholder="Search" autocomplete="off">
```

Comment:
I think `params.searchInputName` comes from the FE template variables which won't be recognised within bloomreach.

Reply:
it is the wrong variable - in the session you and @AlexKirstenZA go through it together, you can correct it to the java variable?